pkg/session_recording/terminal.jsx

```diff
@@ -22,8 +22,11 @@
 
     var React = require("react");
     var Term = require("term");
+    let $ = require("jquery");
 
     require("console.css");
+    require("jquery-resizable");
+    require("jquery-resizable/resizable.css");
```

Comment:
Why I try to run "make" in my cockpit build directory I get this:

    ERROR in /home/nkondras/projects/github.com/cockpit-project/cockpit/pkg/session_recording/terminal.jsx
    Module not found: Error: Cannot resolve module 'jquery-resizable' in /home/nkondras/projects/github.com/cockpit-project/cockpit/pkg/session_recording

Am I missing something?